pkg/volume/iscsi/iscsi.go

```diff
@@ -163,6 +167,7 @@ type iscsiDisk struct {
 	volName string
 	podUID  types.UID
 	portal  string
```

Comment:
Since it is an array now, you can consolidate `portal` and `portals` into `portals`.